src/main/java/com/nextcloud/android/sso/api/AidlNetworkRequest.java

```diff
@@ -76,12 +76,11 @@ public void connect(String type) {
             stop();
         }
 
-        String componentName;
-        if (type != null && type.equalsIgnoreCase(Constants.ACCOUNT_TYPE_DEV)) {
```

Comment:
@tobiasKaminsky The type should never be null and the type is always in lower case, right?